index.html

```diff
@@ -9,7 +9,7 @@
 
 
     <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
-    <link href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
+    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
```

Comment:
I believe it's actually desirable to use // so that the browser uses whichever protocol was used to access the site. (e.g. http, https)